### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by MM1nd**
Some improvements to the looks of the code.

### CODE REVIEW

**Comment by psi29a** [comment]
This looks OK to me, I have a few... 'meh... why?' moments, but nothing that I'm like 'nonononono'.
@ftomassetti any comments?

File: tests/draw_test.py [hunk 1]
```diff
@@ -44,42 +44,45 @@ def _assert_img_equal(self, blessed_image_name, drawn_image):
                         "and/or amount of channels. Blessed %s, drawn %s"
                         % (str(blessed_img.array.shape), str(drawn_image.array.shape)))
 
-        # compare images; cmp_array will be an array of booleans in case of equal shapes (and a pure boolean otherwise)
+        # compare images;
```
Comment (ftomassetti) [inline tests/draw_test.py]: should be ":" instead of ";"?

File: worldengine/biome.py [hunk 2]
```diff
@@ -270,8 +270,9 @@ class TropicalRainForest(Biome, Jungle):
 
 def biome_name_to_index(biome_name):
     names = sorted(_BiomeMetaclass.biomes.keys())
-    for i in range(len(names)):
-        if names[i] == biome_name:
+
```
Comment (ftomassetti) [inline worldengine/biome.py]: maybe we do not need this new extra line

File: worldengine/draw.py [hunk 3]
```diff
@@ -426,7 +431,7 @@ def draw_satellite(world, target):
                             all_b.append(b)
 
                 # Making sure there is at least one valid tile to be smoothed before we attempt to average the values
-                if len(all_r) > 0:
+                if all_r:
```
Comment (ftomassetti) [inline worldengine/draw.py]: are these equivalent?

File: worldengine/generation.py [hunk 4]
```diff
@@ -142,56 +150,55 @@ def harmonize_ocean(ocean, elevation, ocean_level):
...
-    # We want to multiply the raw sea_depth by one of these factors 
+    # We want to multiply the raw sea_depth by one of these factors
     # depending on the distance from the next land
     # possible TODO: make this a parameter
     factors = [0.0, 0.3, 0.5, 0.7, 0.9]
 
     next_land = next_land_dynamic(world.layers['ocean'].data)
 
-    sea_depth = sea_level - world.layers['elevation'].data
+    result = sea_level - world.layers['elevation'].data
```
Comment (ftomassetti) [inline worldengine/generation.py]: I do not like the name result much here

File: worldengine/model/world.py [hunk 5]
```diff
@@ -312,17 +313,19 @@ def _from_protobuf_world(cls, p_world):
...
-        if len(p_world.biome.rows) > 0:
-            w.biome = numpy.array(World._from_protobuf_matrix(p_world.biome, biome_index_to_name), dtype=object)
+        if p_world.biome.rows:
+            w.biome = numpy.array(
+                World._from_protobuf_matrix(p_world.biome, biome_index_to_name), 
+                dtype=object)
 
         # Humidity
-        if len(p_world.humidity.rows) > 0:
+        if p_world.humidity.rows:
```
Comment (ftomassetti) [inline worldengine/model/world.py]: is this equivalent?

**Comment by MM1nd** [comment]
"This looks OK to me, I have a few... 'meh... why?' moments, but nothing that I'm like 'nonononono'."

The answer to "why?" is almost certainly "because pylint said so" and in most cases likely "because the line was too long". I'm agnostic about all of this as witnessed by the three "whitespace" commits at the end of every of my PRs :)

File: tests/draw_test.py (hunk 1 above)
Comment (MM1nd) [reply tests/draw_test.py]: Sure.

File: worldengine/biome.py (hunk 2 above)
Comment (MM1nd) [reply worldengine/biome.py]: Sure.

File: worldengine/draw.py (hunk 3 above)
Comment (MM1nd) [reply worldengine/draw.py]: cf. https://stackoverflow.com/questions/43121340

File: worldengine/generation.py (hunk 4 above)
Comment (MM1nd) [reply worldengine/generation.py]: sea_depth is the name of the function and pylint rightfully complains that this redefines a name from the outer scope. Personally I use `result` a lot as a variable name, but I'm open to anything that is not `sea_depth`. Alternatively the function needs a new name.

File: worldengine/model/world.py (hunk 5 above)
Comment (MM1nd) [reply worldengine/model/world.py]: cf. above

File: worldengine/generation.py (hunk 4 above)
Comment (psi29a) [reply worldengine/generation.py]: depth_of_sea  (in the very french way of saying sea_depth)

**Comment by psi29a** [comment]
I usually set my columns to 120 (from 80)  and adjust linting accordingly...

**Comment by ftomassetti** [comment]
Me too, I use 120 columns and that settings is suggested [here](https://github.com/Mindwerks/worldengine/wiki/Coding-guidelines) (arguably very difficult to find)

**Pull request merged**